Formula/a/ansible-lint.rb

```diff
@@ -182,6 +184,12 @@ def install
   end
 
   test do
+    output = shell_output("#{bin}/ansible-lint --version")
+    ansible_lint_core_version = output.match(/ansible-core:([\d.]+)/)[1]
+
+    ansible = Formula["ansible"].opt_bin/"ansible"
+    assert_match "[core #{ansible_lint_core_version}]", shell_output("#{ansible} --version")
+
```

Comment:
this is the best way I can think of to make sure ansible-core versions in sync between ansible and ansible-lint